syntax = "proto3";
package protos.playbooks;

import "google/protobuf/wrappers.proto";


message Bash {
  message Command {
    google.protobuf.StringValue command = 1;
    google.protobuf.StringValue remote_server = 2;
  }

  enum TaskType {
    UNKNOWN = 0;
    COMMAND = 1;
  }

  TaskType type = 1;
  oneof task {
    Command command = 2;
  }
}